lib/pages/upload/video_handler.dart

```diff
@@ -92,13 +92,13 @@ class _VideoHandlerState extends State<VideoHandler> {
                       _urlController.text; // this will act as our collected url
                   String _name = _nameController.text;
                   addUrl(_url, _name);
-                  Scaffold.of(context)
-                      .showSnackBar(SnackBar(content: Text('Processing data')));
+                  ScaffoldMessenger.of(context).showSnackBar(
+                      const SnackBar(content: Text('Processing data')));
```

Comment:
again?

Reply:
i mean... if i have a reason to change one, surely I have reason to change the others